Provider/src/main/java/com/spotify/confidence/ConfidenceFeatureProvider.kt

```diff
@@ -258,6 +261,7 @@ class ConfidenceFeatureProvider private constructor(
         ): ConfidenceFeatureProvider {
             val configuredClient = client ?: ConfidenceRemoteClient(
                 clientSecret = clientSecret,
+                sdkMetadata = SdkMetadata(SDK_ID, BuildConfig.SDK_VERSION),
```

Comment:
Doesn't work, the result string for version is: `Build_gradle$Versions@1e4d110c.providerVersion`. To be fixed